releasenotes/notes/fix-required-memory-check-0a138e0478d8ff32.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Fixes issue where memory requirements of simulation were not being checked
+    on the QasmSimulator when using a non-automatic simulation method.
```

Comment:
I thought this was automatically generated when creating the release?

Reply:
I thought you still needed these since they have them in Terra, but maybe @mtreinish knows?

Reply:
Yes this is still needed. The changelog is automated now, but release documentation still needs to be written. The previous changelog was serving double duty as release documentation (frankly quite poorly). See the contributing docs on this: https://github.com/Qiskit/qiskit-aer/blob/master/CONTRIBUTING.md#release-notes (the changelog section above it mentions the changelog autogeneration)